mujoco_warp/_src/types.py

```diff
@@ -1732,3 +1732,45 @@ class Data:
 
   # actuator
   actuator_trntype_body_ncon: wp.array2d(dtype=int)  # warp only
+
+
+class State(enum.IntEnum):
```

Comment:
nit: let's keep the enums all grouped together up at the top

Reply:
done